Components/Card/Card.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="Card.css">
+  <title>Perry's Card</title>
+</head>
+
+<body>
+
+<div id="card">
```

Comment:
What is the difference between an ID and a Class in CSS? Why is it not a good idea to use an ID here?

Reply:
Because I might want to apply the styles to more cards